package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "tsc",
     "prestart": "npm run build",
-    "now-build": "yum install gcc-c++ cairo-devel pango-devel libjpeg-turbo-devel giflib-devel && npm run build",
+    "now-build": "yum install gcc-c++ cairo-devel pango-devel libjpeg-turbo-devel giflib-devel libuuid-devel libmount-devel && cp /lib64/{libuuid,libmount,libblkid}.so.1 node_modules/canvas/build/Release/ && npm run build",
```

Comment:
Why is this change needed?

Reply:
Yea somehow there was an error in Now deployment that didn't have `uuid` so I need to install those extra packages

Reply:
That doesn't look right to me. It wasn't required in the past, right?

Reply:
Yea it wasn't. This is just a problem with Now deployment. I followed the open issue [here](https://github.com/vercel/vercel/issues/3460) and copy the command.